src/views/RoomDetail.vue

```diff
@@ -49,17 +64,92 @@ export default {
   created () {
     this.roomID = this.$route.params.roomID
   },
-  mounted () {
-    const db = this.$firebase.firestore()
-    db
-      .collection('Rooms')
-      .doc(this.roomID)
-      .get()
-      .then(doc => {
-        this.room = doc.data()
-      })
+  async mounted () {
+    await this.getRoomKeyValuePairs()
+    await this.getRoomCheckIns()
+    await this.getRoomCheckInsTomorrow()
+    await this.fetchUserIDList()
+    await this.isUserSignedIn()
+    await this.isUserSignedInTomorrow()
+    await this.roomCapacityToday()
+    await this.roomCapacityTomorrow()
   },
   methods: {
+    getRoomKeyValuePairs: async function () {
+      await $db()
+        .collection('Rooms')
+        .doc(this.roomID)
+        .get()
+        .then(doc => {
+          this.room = doc.data()
+          this.room.checkIns = []
+          this.room.checkInsTomorrow = []
+        })
+    },
+    getRoomCheckIns: async function () {
+      await $db()
+        .collection('Rooms')
+        .doc(this.roomID)
+        .collection('CheckIn')
```

Comment:
diesen Aufruf kann man zusammenfassen mit
```suggestion
        await $db().collection('Rooms/' + this.roomID + '/CheckIn')
```

Reply:
Finde ich so übersichtlicher.

Reply:
Ok, performance technisch macht das keinen Unterschied, es spart nur Codezeilen. Für größere Methoden ist das praktisch, da wir ja die URL der Collection nicht ändern werden.

Reply:
In Teams geklärt. 👍